ibm/resource_ibm_is_virtual_endpoint_gateway_ip.go

```diff
@@ -0,0 +1,213 @@
+package ibm
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+const (
+	isVirtualEndpointGatewayID                   = "gateway_id"
```

Comment:
use gateway instead  of gateway_id to remain in sync with all VPC reosurces

Reply:
handled